### PR TITLE
website/integrations: add Jenkins docs

### PR DESCRIPTION
**Pull request by BeryJu**
## Details

REPLACE ME

---

## Checklist

-   [ ] Local tests pass (`ak test authentik/`)
-   [ ] The code has been formatted (`make lint-fix`)

If an API change has been made

-   [ ] The API schema has been updated (`make gen-build`)

If changes to the frontend have been made

-   [ ] The code has been formatted (`make web`)
-   [ ] The translation files have been updated (`make i18n-extract`)

If applicable

-   [ ] The documentation has been updated
-   [ ] The documentation has been formatted (`make website`)

### CODE REVIEW

File: website/integrations/services/dokuwiki/index.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ The following placeholders will be used:
 -   `dokuwiki.company` is the FQDN of the DokiWiki install.
 -   `authentik.company` is the FQDN of the authentik install.
 
-## Service Configuration
+## DokuWiki Configuration
```
Comment (tanberry) [inline website/integrations/services/dokuwiki/index.md:20]:
```suggestion
## DokuWiki configuration
```

File: website/integrations/services/jellyfin/index.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ The following placeholders will be used:
 -   `dc=company,dc=com` the Base DN of the LDAP outpost.
 -   `ldap_bind_user` the username of the desired LDAP Bind User
 
-## Service Configuration
+## Jellyfin Configuration
```
Comment (tanberry) [inline website/integrations/services/jellyfin/index.md:35]:
```suggestion
## Jellyfin configuration
```

File: website/integrations/services/jenkins/index.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+---
+title: Jenkins
+---
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is Jenkins
+
+> The leading open source automation server, Jenkins provides hundreds of plugins to support building, deploying and automating any project.
```
Comment (tanberry) [inline website/integrations/services/jenkins/index.md:9]:
```suggestion
> Jenkins is open source automation server, with hundreds of plugins to support building, deploying, and automating any project.
```

File: website/integrations/services/jenkins/index.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+> -- https://www.jenkins.io/
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `jenkins.company` is the FQDN of the Service install.
+-   `authentik.company` is the FQDN of the authentik install.
+
+Create an application in authentik. Create an OAuth2/OpenID provider with the following parameters:
```
Comment (tanberry) [inline website/integrations/services/jenkins/index.md:20]: would be good to link to our docs on how to create a provider.

File: website/integrations/services/jenkins/index.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+-   Client Type: `Confidential`
```
Comment (tanberry) [inline website/integrations/services/jenkins/index.md:22]:
```suggestion
-   **Client Type**: `Confidential`
```

File: website/integrations/services/jenkins/index.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+-   Scopes: OpenID, Email and Profile
```
Comment (tanberry) [inline website/integrations/services/jenkins/index.md:23]:
```suggestion
-   **Scopes**: OpenID, Email and Profile
```

File: website/integrations/services/jenkins/index.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+-   Signing Key: Select any available key
```
Comment (tanberry) [inline website/integrations/services/jenkins/index.md:24]:
```suggestion
-   **Signing Key**: Select any available key
```

File: website/integrations/services/jenkins/index.md (hunk 4 above)
Comment (tanberry) [inline website/integrations/services/jenkins/index.md:20]:
```suggestion
Create an OAuth2/OpenID provider with the following parameters:
```

Comment (tanberry) [reply website/integrations/services/jenkins/index.md:20]: I'd suggest you first have them create the provider, and then below create the app.

File: website/integrations/services/jenkins/index.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+Note the Client ID and Client Secret values. Create an application, using the provider you've created above. Note the slug of the application you've created.
```
Comment (tanberry) [inline website/integrations/services/jenkins/index.md:26]:
```suggestion
Note the Client ID and Client Secret values for the provider. 

Next, create an application, using the provider you've created above. Note the slug of the application you create.
```

File: website/integrations/services/jenkins/index.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+
+## Jenkins Configuration
+
+Navigate to the Jenkins plugin manager by navigating to **Manage Jenkins** then **Plugins** then **Available plugins**. Search for the plugin `oic-auth` in the search field, and install the plugin. Jenkins must be restarted afterwards to ensure the plugin is loaded.
```
Comment (tanberry) [inline website/integrations/services/jenkins/index.md:30]:
```suggestion
Navigate to the Jenkins plugin manager: **Manage Jenkins** -> **Plugins** -> **Available plugins**. Search for the plugin `oic-auth` in the search field, and install the plugin. Jenkins must be restarted afterwards to ensure the plugin is loaded.
```

File: website/integrations/services/jenkins/index.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+
+After the restart, navigate to **Manage Jenkins** again, then clicking on **Security**.
```
Comment (tanberry) [inline website/integrations/services/jenkins/index.md:32]:
```suggestion
After the restart, navigate to **Manage Jenkins** again, and click **Security**.
```

File: website/integrations/services/jenkins/index.md [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+
+Modify the **Security Realm** option and select `Login with Openid Connect`.
```
Comment (tanberry) [inline website/integrations/services/jenkins/index.md:34]:
```suggestion
Modify the **Security Realm** option to select `Login with Openid Connect`.
```

File: website/integrations/services/jenkins/index.md [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+
+Fill the Client ID and Client Secret values from above into the respective **Client id** and **Client secret** fields.
```
Comment (tanberry) [inline website/integrations/services/jenkins/index.md:36]:
```suggestion
In the **Client id** and **Client secret** fields, enter the Client ID and Client Secret values from the provider you created.
```

File: website/integrations/services/jenkins/index.md [hunk 13]
```diff
@@ -0,0 +1,51 @@
...
+Check the checkbox **Override scopes** and input the scopes `openid profile email` into the new input field.
+
+Further down the page, expand the **Advanced** section and input the following values:
+
+-   **User name field name**: `preferred_username`
+-   **Full name field name**: `name`
+-   **Email field name**: `email`
+-   **Groups field name**: `groups`
+
+It is also recommended to enable the option **Enable Proof Key for Code Exchange** further down the page.
```
Comment (tanberry) [inline website/integrations/services/jenkins/index.md:49]:
```suggestion
We also recommend enabling the option **Enable Proof Key for Code Exchange** further down the page.
```

File: website/integrations/services/jenkins/index.md [hunk 14]
```diff
@@ -0,0 +1,51 @@
...
+
+Additionally, as a fallback to regain access to Jenkins in the case of misconfiguration, it is recommended to configure the **Configure 'escape hatch' for when the OpenID Provider is unavailable** option below. How to configure this option is beyond the scope of this document, and is explained by the OpenID Plugin.
```
Comment (tanberry) [inline website/integrations/services/jenkins/index.md:51]:
```suggestion
Additionally, as a fallback to regain access to Jenkins in the case of misconfiguration, we recommend configuring the **Configure 'escape hatch' for when the OpenID Provider is unavailable** option below. How to configure this option is beyond the scope of this document, and is explained by the OpenID Plugin.
```

**Review by tanberry — APPROVED** [review]
Lots of nitpicks, which I can add to our template, but yeah, LGTM!

File: website/integrations/services/jenkins/index.md (hunk 3 above)
Comment (BeryJu) [reply website/integrations/services/jenkins/index.md:9]: this is taken 1:1 from their webpage so I'll keep it as is

**Pull request merged**